backend/src/article/utils/checkValidate.ts

```diff
@@ -0,0 +1,16 @@
+export const checkArticleValidate = (plans: string[]) => {
+  if (!plans) {
+    return false;
+  }
+
+  if (plans.length > 10) {
+    return false;
+  }
+
+  for (let i = 0; i < plans.length; i++) {
+    if (plans[i].length < 1) {
+      return false;
+    }
+  }
+  return true;
+};
```

Comment:
저 아무래도 클린코드 강박증이 있나봐요.. 극한의 다이어트....

```suggestion
export const checkArticleValidate = (plans: string[]) => {
  return plans && plans.length <= 10 && plans.findIndex(plan => plan.length < 1) < 0;
};
```

Reply:
편안하네요... 전 네이밍 강박증이 있나봐요....

```suggestion
export const checkArticleValidate = (plans: string[]) => {
  return plans && plans.length <= ARTICLES_PER_PAGE && plans.findIndex(plan => plan.length < 1) < 0;
};
```

Reply:
ㅋㅋㅋㄱㅋㄱㅋㅋㄱㅋㄷ감사합니다 여러분
앉아서 냠냠하겠습니당😘